SCF-Session-6/polynomial/PolyTest.java

```diff
@@ -0,0 +1,95 @@
+package polynomial;
+
+import static org.junit.Assert.*;
+
+public class PolyTest {
+
+    /**
+     * Test case when input polynomial is null
+     */
+    @org.junit.Test(expected = AssertionError.class)
+    public void polyFirstException() 
+    {
+        new Poly(null);
```

Comment:
Exception message should be asserted in test methods if exception is expected.

Reply:
done